tests/profile/update/loginWithOldPassword.spec.js

```
@@ -1,31 +1,26 @@
 import { test } from '../../_fixtures/fixtures';
-import { signUpUser } from '../../../src/ui/actions/auth/signUpUser';
 import { INVALID_EMAIL_OR_PASSWORD_MESSAGE } from '../../../src/ui/constants/authErrorMessages';
+import { SignInPage } from '../../../src/ui/pages/auth/SignInPage';
 
-let newPassword;
+test('Login with old password after it was updated via API', async ({
+  page,
+  registeredUser,
+  factories,
+  usersApi,
+}) => {
+  const signInPage = new SignInPage(page);
 
-test.beforeEach(async ({ page, user, factories }) => {
-  await signUpUser(page, user);
+  const newPassword = factories.user.generatePassword();
 
-  newPassword = factories.user.generatePassword();
-});
+  const updatedUser = { ...registeredUser, password: newPassword };
+  await usersApi.updateUser(updatedUser);
```

The call to `usersApi.updateUser(updatedUser)` doesn't capture or assert the API response. Capture the response and assert success (for example `const response = await usersApi.updateUser(...); expect(response.ok()).toBeTruthy();`) so the test fails early if the update didn't apply.